packages/neon/neon/test/timer_bloc_test.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:neon/neon.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('TimerBloc', () {
+    test('Register timer', () async {
+      const duration = Duration(milliseconds: 100);
+
+      final stopwatch = Stopwatch()..start();
+      TimerBloc().registerTimer(duration, stopwatch.stop);
+      await Future.delayed(duration);
+
+      expect(stopwatch.elapsedMilliseconds, greaterThan(duration.inMilliseconds));
+      expect(stopwatch.elapsedMilliseconds, lessThan(duration.inMilliseconds * 1.1));
```

Comment:
We might want to verify that we actually store the data:

```dart
    test('Register timer', () async {
      const duration = Duration(milliseconds: 100);

      final stopwatch = Stopwatch()..start();
      final callback = stopwatch.stop;
      TimerBloc().registerTimer(duration, callback);
      await Future.delayed(duration);

      expect(stopwatch.elapsedMilliseconds, greaterThan(duration.inMilliseconds));
      expect(stopwatch.elapsedMilliseconds, lessThan(duration.inMilliseconds * 1.1));
      expect(TimerBloc().callbacks[duration.inSeconds], contains(callback));
      expect(TimerBloc().timers[duration.inSeconds], isNot(isNull));
    });
```

This requires the underlying `TimerBloc.timers` and `TimerBloc.callbacks` to not have an underscore and be `@VisibleForTesting`